via/static/scripts/video_player/components/test/TranscriptError-test.js

```diff
@@ -0,0 +1,33 @@
+import { mount } from 'enzyme';
+
+import { APIError } from '../../utils/api';
+import TranscriptError from '../TranscriptError';
+
+describe('TranscriptError', () => {
+  it('displays just `Error.message` if there are no error details', () => {
+    const wrapper = mount(
+      <TranscriptError error={new Error('Something went wrong')} />
+    );
+    assert.equal(
+      wrapper.text(),
+      ['Unable to load transcript', 'Something went wrong'].join('')
```

Comment:
`[...].join('')` is used here to separate out parts that are on different lines in the UI, but that don't have actual spaces between them.